Utilities/DataLoading/EdFi.XmlLookup.Console/Application/CommandLineOverrides.cs

```diff
@@ -41,7 +41,7 @@ public class CommandLineOverrides
         [Option('w', "working", Required = false, HelpText = "Path to a writable folder containing the working files")]
         public string WorkingFolder { get; set; }
 
-        [Option('x', "xsd", Required = false, HelpText = "Path to a folder containing the Ed-Fi Xsd Schema files")]
+        [Option('x', "xsd", Required = false, HelpText = "(deprecated) Path to a folder containing the Ed-Fi Xsd Schema files")]
         public string XsdFolder { get; set; }
 
         [Option('b', "baseUrl", Required = false, HelpText = "The base url")]
```

Comment:
@raghuramakrishna while you are at it, could you bring baseUrl to the top of this list so that is shows first when --help is called